pytest_sugar.py

```diff
@@ -398,7 +399,7 @@ def pytest_runtest_logstart(self, nodeid, location):
         # show the module_name & in verbose mode the test name.
         pass
 
-    if pytest.__version__ >= '3.4':
+    if parse(pytest.__version__) >= parse('3.4'):  # pragma: no cover
```

Comment:
I don't like the way the code is written with the "current" version being marked as the deviant behavior. I'd write the code differently:

```
    def pytest_runtest_logfinish(self):
        """prevent the default implementation to try to show
        pytest's default progress"""

    if parse(pytest.__version__) < parse(3.4):  # pragma: nocover
        # On older Pytest, allow default progress
        del pytest_runtest_runfinish
```

Reply:
What would be the harm of including this clause unconditionally, just like its sister method above?

Reply:
Yes it makes sense, I will do that.